data/strategies/42161/CurveGauge.json

```diff
@@ -19,7 +19,7 @@
     },
     "de": {
       "name": "Curve Gauge",
-      "description": "Supplies {{token}} to [Curve Finance](https://arbitrum.curve.fi) and stakes it in gauge to collect any available tokens and earn CRV rewards. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy."
+      "description": "Liefert {{Token}} an [Curve Finance](https://arbitrum.curve.fi) und setzt sie in der Lehre ein, um alle verfügbaren Token zu sammeln und CRV-Belohnungen zu verdienen. Verdiente Token werden geerntet und für weitere {{Token}} verkauft, die dann wieder in die Strategie eingezahlt werden."
```

Comment:
dont capitalize token

Reply:
In German the word "token" is a noun and should thus be capitalized. Do I always have to leave to like that if it is in brackets?

Reply:
i have changed in bracket to small letters.